File: get_feature_51.py
import sqlite3
import json

conn = sqlite3.connect('features.db')
conn.row_factory = sqlite3.Row
cursor = conn.cursor()

cursor.execute('SELECT * FROM Features WHERE Id = 51')
row = cursor.fetchone()

if row:
    print(json.dumps(dict(row), indent=2))
else:
    print("Feature #51 not found")

conn.close()

File: get_feature_51.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('./features.db');

db.get('SELECT * FROM Features WHERE Id = 51', (err, row) => {
  if (err) {
    console.error(err);
  } else {
    console.log(JSON.stringify(row, null, 2));
  }
  db.close();
});
